loadtests/loadtest.py

```diff
@@ -0,0 +1,19 @@
+from molotov import scenario
+
+
+HOMEPAGE = 'https://fx-breach-alerts.herokuapp.com'
+SCAN = 'https://fx-breach-alerts.herokuapp.com/scan'
+# This hash is for test@example.com
+TEST_EMAIL_HASH = 'b84c4c03b2af4050ac2d3b105e58adf83fa5af05'
```

Comment:
Your way of generating the hash was correct: this hash is not the correct hash of test@example.com. Could you update this please? Thanks for catching this :)